docs/cloud-native-security/cloud-native-security-index.asciidoc

```diff
@@ -0,0 +1,7 @@
+[[cloud-native-security-overview]]
+
+= Cloud native security
+Elastic’s cloud security capabilities help you to improve your Kubernetes security posture by comparing your configuration to best practices, and help you monitor and investigate your Linux deployments inside and outside of Kubernetes. This is an area of focus for Elastic Security, and you can expect many more cloud security features in future releases.
```

Comment:
The second sentence sounds more like a blog or marketing statement, and even though it's pretty general we try to avoid forward-looking statements that seem to "promise" things.

I know the focus of this PR is more about structure than content, so we can do a closer review later if there's more to come for this section.

Reply:
I would definitely nix the second sentence.